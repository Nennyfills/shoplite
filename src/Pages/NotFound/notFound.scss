.not-found-page {
  background-image: url('../../Assets/images/notFound.jpeg');
  width: 100%;
  height: 100%;
  object-fit: cover;
  background-repeat: no-repeat;
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-position: center;
}
