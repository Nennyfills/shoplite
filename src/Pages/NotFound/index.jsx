import React from 'react';

import './notFound.scss';

const NotFound = () => {
  return (
    <div className="not-found-page" data-test="not-found-page">
      NotFound
    </div>
  );
};

export default NotFound;
