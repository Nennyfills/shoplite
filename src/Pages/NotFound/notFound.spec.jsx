import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '.';

const setUp = (props = {}) => {
  const component = shallow(<NotFound {...props} />);
  return component;
};

describe('NotFound Page', () => {
  let component;
  beforeEach(() => {
    component = setUp({});
  });
  it('Should render without errors', () => {
    expect(component).toMatchSnapshot();
  });
  it('Should render Not Found Page div', () => {
    const wrapper = component.find(`[data-test='not-found-page']`);
    expect(wrapper.length).toBe(1);
  });
});
