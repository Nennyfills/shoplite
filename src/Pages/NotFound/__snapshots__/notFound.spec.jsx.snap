// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Page Should render without errors 1`] = `ShallowWrapper {}`;
